pages/reference/OS/advanced.md

```diff
@@ -5,94 +5,91 @@ excerpts: Configuration options to expose more device functionality to {{ $names
 
 # Advanced boot settings
 
-__Warning:__ This page contains details of the more *involved* configuration
-options that expose more functionality but potentially enable you to break
-things should you configure them incorrectly - tread carefully!
+__Warning:__ This page contains details of advanced configuration options that expose more functionality, but any mistakes may potentially leave a device inaccessible.
 
 ## Raspberry Pi
 
-### config.txt
+The Raspberry Pi exposes device [configuration options][config-txt] via a text file on the [boot partition][boot-partition] named [`config.txt`][config-txt]. You can change boot options simply by editing this file, either by manually editing before the device's first boot or editing the device [configuration variables][config-vars]. By default, the following values are set for Raspberry Pi devices:
 
-The Raspberry Pi exposes device configuration options via a text file on the
-boot medium, `config.txt` - you change boot options simply by editing this file.
+{{> "general/config-variables-pi" }}
 
+The boot partition is mounted on the device at `/mnt/boot`, and so on the device, the file is located at `/mnt/boot/config.txt`. For example, to view the contents of `config.txt` on a provisioned device use the following commands:
 
-#### Modifying `config.txt` locally before the first boot
+```shell
+$ balena ssh <uuid>
+$ cat /mnt/boot/config.txt
+```
 
-The `config.txt` is located in the root of the `{{ $names.company.lower }}-boot` partition, and you can modify it by mounting the SD card on a computer. This will only work if you edit the SD card before the device's first boot.
+### Modifying `config.txt` locally before the first boot
 
+Before the device is [provisioned][device-provisioning], you may edit `config.txt` by mounting a flashed SD card (with the partition label `resin-boot`) and editing the file directly. This will only work if you edit the file before the device's first boot as after device provisioning, any changes will be overwritten by the device supervisor with values read from the {{ $names.cloud.lower }} API.
```

Comment:
I think it's worth clarifying a bit further here, added a suggestion but feel free to reword:

```suggestion
Before the device is [provisioned][device-provisioning], you may edit `config.txt` by mounting a flashed SD card (with the partition label `resin-boot`) and editing the file directly. This will only work if you edit the file before the device's first boot as after device provisioning, any changes will be overwritten by the device supervisor with values read from the {{ $names.cloud.lower }} API. For this reason if you wish changes added before the first boot to persist, they must also be specified in the dashboard.
```

Reply:
Yes definitely, I wasn't aware of that. I assumed they would be added during provisioning.

Reply:
Aha! Reading the old version of this page, it sounded like the supervisor just "appended" new values to config.txt, but I guess you're saying that the supervisor just overwrites the entirety of config.txt?

Reply:
@chrisys I'm confused by this comment as I just tested this and works as initially outlined but I might have missed something.

To test, I simply created a new application and added a new device (Raspberry Pi 4). I flashed the image with Etcher and edited `config.txt` before the first boot. I added `start_x=1` and then provisioned the device. As expected a custom configuration value was already created for me in the dashboard, so they are persisted so long as they are created prior to device provisioning.

Reply:
I tested again on a production image with some more obscure values to double check and they get added to the dashboard automatically if you do this prior to device provisioning. 

<img width="1121" alt="CleanShot 2020-03-02 at 10 28 37@2x" src="https://user-images.githubusercontent.com/135382/75705630-9985d980-5c70-11ea-84c4-b499e40c0ff6.png">

Reply:
@lurch so in summary, as you point out, yes the supervisor does overwrite the file rather than append but if you add these before the device is provisioned these are created in the dashboard (API) so the supervisor is aware of them so they won't be removed. If you edit `config.txt` after provisioning any changes you make are simply going to get overwritten by the supervisor so you must use the dashboard.

Reply:
If that's indeed the case, it's probably worth explicitly mentioning in the docs that any values manually added to config.txt before device-provisioning are also automatically added to the dashboard when the device is provisioned?